android/src/main/java/co/nimblehq/kmm/template/android/ui/navigation/AppNavigation.kt

```diff
@@ -0,0 +1,44 @@
+package co.nimblehq.kmm.template.android.ui.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.*
+import androidx.navigation.compose.*
+import co.nimblehq.kmm.template.android.ui.screens.home.HomeScreen
+
+@Composable
+fun AppNavigation(
+    navController: NavHostController = rememberNavController(),
+    startDestination: String = AppDestination.Home.destination
+) {
+    NavHost(
+        navController = navController,
+        startDestination = startDestination
+    ) {
+        composable(AppDestination.Home) {
+            HomeScreen(
+                // TODO
```

Comment:
Shall we add some sentence here?

Reply:
@kaungkhantsoe Yes, however, let's add more initial examples in the following tasks where we added an initial MVVM flow from data > domain > app level.